README.md

```
@@ -25,4 +23,56 @@ Please note that the README doesn't affect your grade and is not included in doc
 
 ### Installation
 
-[Installation link here]
+### Prerequisites
+- **Docker** & **Docker Compose**
+- **Android Studio**: Iguana or newer, **JDK 17**
+- **Android SDK**: minSdk 24+
+
+### Installation
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)